starters/react-ts/src/client/pages/index.tsx

```diff
@@ -0,0 +1,8 @@
+export default function Index() {
```

Comment:
nit: Usually React entry point applications like to name their root component `<App>`, not `<Index>`

Reply:
This is a page component. The root component is `Root`. See [here](https://github.com/fastify/fastify-vite/blob/main/packages/fastify-react/virtual/create.jsx).